python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -51,38 +59,28 @@ def __init__(
         when serializing data.
 
         Args:
-            writer: The actor that may write to the channel. None signifies the driver.
+            writer: The actor that may write to the channel. None signifies the
+                driver.
             reader_and_node_list: A list of tuples, where each tuple contains a reader
                 actor handle and the node ID where the actor is located.
-            gpu_data_typ: Type information about the GPU tensors
-            cpu_data_typ: Type information about the CPU data
+            tensor_data_channel: A GPU-GPU channel for sending tensor data. Its
+                writer and readers should match the given writer and readers.
+            non_tensor_data_channel: A shared-memory channel for sending
```

Comment:
I think this is technically wrong? I think cpu tensors are still sent through this channel.